packages/minifier-js/minifier.js

```diff
@@ -29,14 +30,11 @@ meteorJsMinify = function (source) {
     // Although Babel.minify can handle a wider variety of ECMAScript
     // 2015+ syntax, it is substantially slower than UglifyJS, so we use
     // it only as a fallback.
-    if (Babel.getMinifierOptions) {
```

Comment:
Why did you remove this check?

Reply:
Since getMinifierOptions is supported since meteor-babel 0.16, this check seems very outdated. I could leave it in, but then I would need some way of communicating back whether options are passed to Babel or not. IMO, babel-minify must always be configured the same as uglify, because otherwise using the fallback can cause unexpected failures that don't happen when the fallback isn't used. Therefore, I think that the options must always be passed to babel-minify.